api/src/auth/drivers/oauth2.ts

```diff
@@ -47,11 +48,18 @@ export class OAuth2AuthDriver extends LocalAuthDriver {
 			issuer: additionalConfig.provider,
 		});
 
+		const clientOptionsOverrides = getConfigFromEnv(
+			`AUTH_${config.provider.toUpperCase()}_CLIENT_`,
+			[`AUTH_${config.provider.toUpperCase()}_CLIENT_ID`, `AUTH_${config.provider.toUpperCase()}_CLIENT_SECRET`],
+			'underscore'
+		);
```

Comment:
@aidenfoxx You're already extracting the `additionalConfig` in here:

https://github.com/directus/directus/blob/f8fa3a4cd3a73485b5eb9850f921f292b97764c7/api/src/auth.ts#L47-L48

Howeverrrrrrrrrrrrrr that's `camelCased`, and not scoped to just CLIENT_.. Wasn't quite sure what would be the cleanest place to put this 🤔 I figured seeing it's very openid/oauth2 specific, and not so much a global "additionaloptions" thing we can just pass in, it would be OK to just do it in here. Lemme know your thoughts!

Reply:
I think it does the job fine. 👍